foia_hub/static/js/main.js

```diff
@@ -41,9 +42,17 @@ $(document).ready(function(){
         }
     }
     $bannerCloseButton.click(function(){
-        $("#notice").slideUp();
+        $("#notice").slideUp(function () {
+          times_toggled++
+            ga('send', {
```

Comment:
Descriptive IDs have been added to the expandable list items. Should those get passed through as well so that we know which definition was expanded? Or does that happen automatically?